src/browser/main.coffee

```diff
@@ -69,7 +69,9 @@ setupCrashReporter = ->
 setupAtomHome = ->
   return if process.env.ATOM_HOME
 
-  atomHome = path.join(app.getHomeDir(), '.atom')
+  atomHome = path.join(path.dirname(process.execPath), 'userdata')
```

Comment:
>  it doesn't have the "Update.exe", without this Atom will not auto-update, right?

Yes, it won't auto update.

> We can change the path to '..\userdata', so it will stay with the updater and will not accidentally erased?

I'm not sure exactly what you mean here, sorry, are you talking about looking for the `userdata` at `C:\Users\kevin\AppData\Local\atom\userdata`?